build-system/pr-check.js

```diff
@@ -255,6 +255,22 @@ function determineBuildTargets(filePaths) {
   return targetSet;
 }
 
+function startSauceConnect() {
+  process.env['SAUCE_ACCESS_KEY'] = getStdout(
+      'curl --silent https://amphtml-sauce-token-dealer.appspot.com/getToken')
```

Comment:
Where is the code for `https://amphtml-sauce-token-dealer.appspot.com/getToken`?

Reply:
Replied offline.